<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Aplikasi_Forensik</class>
 <widget class="QMainWindow" name="Aplikasi_Forensik">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="scanButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>17</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>17</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>400</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Oke</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/images/checkIcon.png</normaloff>:/images/images/checkIcon.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="extractButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>400</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Extract</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/images/extractIcon.png</normaloff>:/images/images/extractIcon.png</iconset>
    </property>
   </widget>
   <widget class="QGroupBox" name="Input">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>401</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
    <widget class="QLabel" name="fileName">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>42</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>File Name   :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="search">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>35</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="searchFile">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>45</y>
       <width>27</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
    </widget>
    <widget class="QLabel" name="examiner">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Examiner    :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>75</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="outputLocation">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>120</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Output Location :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="browseLocation">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>114</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="browse">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>117</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="acquisitionBar">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>350</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="acquisitionProgress">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>353</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Acquisition Progress    :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="detectDevice">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>Redmi Note 4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Samsung Galaxy J1 Ace</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Redmi Note 8</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="resultCek">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>54</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result :</string>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>55</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Not Root!</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="destinationFile_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>100</y>
      <width>421</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QLabel" name="imageType">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>File Type and Storage  : </string>
     </property>
    </widget>
    <widget class="QRadioButton" name="text">
     <property name="geometry">
      <rect>
       <x>203</x>
       <y>33</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>.txt</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="internal">
     <property name="geometry">
      <rect>
       <x>271</x>
       <y>31</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Local</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="detailData">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>63</y>
       <width>371</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Details</string>
     </property>
     <widget class="QListView" name="viewData">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>20</y>
        <width>351</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_App"/>
    <addaction name="actionAbout_QT"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/folderIcon.png</normaloff>:/images/images/folderIcon.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/quitIcon.png</normaloff>:/images/images/quitIcon.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout_App">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/aboutIcon.png</normaloff>:/images/images/aboutIcon.png</iconset>
   </property>
   <property name="text">
    <string>About App</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/aboutQtIcon.png</normaloff>:/images/images/aboutQtIcon.png</iconset>
   </property>
   <property name="text">
    <string>About QT</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
